src/common/dom/load_resource.ts

```diff
@@ -22,6 +22,7 @@ const _load = (
         (element as HTMLScriptElement).async = true;
         if (type) {
           (element as HTMLScriptElement).type = type;
+          (element as HTMLScriptElement).crossOrigin = "use-credentials";
```

Comment:
Please add a comment that links back to this PR
```suggestion
          // https://github.com/home-assistant/frontend/pull/6328
          (element as HTMLScriptElement).crossOrigin = "use-credentials";
```